.github/workflows/test.yml

```diff
@@ -0,0 +1,29 @@
+name: Tests
+
+on: 
+  pull_request:
+  workflow_dispatch:
+  push:
+    branches: [main]
+
+jobs: 
+  unit-test:
+    name: Tests
+    runs-on: ubuntu-latest
+    steps: 
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with: 
+          node-version: "14"
```

Comment:
Lets take the node versions from a matrix.

Ref: https://github.com/asgardeo/asgardeo-auth-react-sdk/blob/main/.github/workflows/pr-builder.yml#L32

Also, lets use `LTS` version.